components/brave_rewards/browser/database/database_unblinded_token.cc

```diff
@@ -170,10 +170,10 @@ bool DatabaseUnblindedToken::MigrateToV15(sql::Database* db) {
   return true;
 }
 
-bool DatabaseUnblindedToken::InsertOrUpdate(
+bool DatabaseUnblindedToken::InsertOrUpdateList(
```

Comment:
This was changed to reduce potential error where not all tokens are inserted. That's why now we insert them all in one transaction.